pulp_smash/tests/rpm/api_v2/test_upload_publish.py

```diff
@@ -149,6 +154,59 @@ def _get_pkglist(update):
     return out_pkglist
 
 
+class UploadDrpmTestCase(utils.BaseAPITestCase):
+    """Test whether one can upload and associate DRPMs.
+
+    Steps:
+
+    1. Create a RPM repo.
+    2. Upload DRPM to repo.
+    3. Check if DRPM has been uploaded.
+
+
+    This test case targets
+    `Pulp Smash #336 <https://github.com/PulpQE/pulp-smash/issues/336>`_
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Upload DRPM to a repo."""
+        super(UploadDrpmTestCase, cls).setUpClass()
+
```

Comment:
This test fails when run against Pulp systems, regardless of whether they've been installed from source, the nightly RPMs, or from official RPMs. Something like this will conditionally skip the test:

```python
if selectors.bug_is_untestable(1806, cls.cfg.version):
    raise unittest2.SkipTest('https://pulp.plan.io/issues/1806')
```